actors/cloudbroker/cloudbroker__computenode/methodclass/cloudbroker_computenode.py

```diff
@@ -152,8 +152,8 @@ def maintenance(self, id, vmaction, **kwargs):
         :return: bool
         """
         title = ""
-        if vmaction not in ('move', 'stop'):
-            raise exceptions.BadRequest("VMAction should either be move or stop")
+        if vmaction not in ('move', 'stop', 'migrate'):
```

Comment:
how is this different from move, ( i understand this is for the demo but it should not be on master )